api/src/pcapi/core/bookings/api.py

```diff
@@ -102,6 +103,16 @@ def _is_ended_booking(booking: Booking) -> bool:
     )
 
 
+def is_booking_by_18_user(booking: Booking) -> bool:
+    if not booking.deposit:
+        return False
+    if booking.deposit.type == finance_models.DepositType.GRANT_18:
+        return True
+    if booking.deposit.type == finance_models.DepositType.GRANT_17_18:
+        return get_age_at_date(booking.user.birth_date, booking.dateCreated, booking.user.departementCode) <= 18
```

Comment:
```suggestion
        return get_age_at_date(booking.user.birth_date, booking.dateCreated, booking.user.departementCode) >= 18
```
ou alors j'ai pas compris le nom de la fonction ?
une autre manière de faire serait d'appeler `has_been_recredited(user, age=18)`

Reply:
Oula oui !
ça veut dire que tous les tests sont pour des == 18, faut que j'ajoute un test en +